tests/function_modifiers/test_adapters.py

```diff
@@ -367,7 +367,7 @@ def test_loader_fails_for_missing_attribute():
 
 
 def test_pandas_extensions_end_to_end(tmp_path_factory):
-    output_path = tmp_path_factory.mktemp("test_pandas_extensions_end_to_end") / "output.csv"
```

Comment:
Posix paths are fine, but I didn't want to change everything to have a union now, so casting is just as easy.